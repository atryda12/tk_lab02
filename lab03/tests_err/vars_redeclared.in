
int x=0,y=0,z=0;
string s="";
int u=0,v=0,x=0;

print x;

