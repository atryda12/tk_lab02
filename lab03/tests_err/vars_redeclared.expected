
Error: Variable 'x' already declared: line 4

